renku/service/jobs/datasets.py

```diff
@@ -75,6 +75,7 @@ def dataset_import(
     dataset_uri,
     short_name=None,
     extract=False,
+    timeout=None,
```

Comment:
This is not used anywhere. Is there a reason for that?

Reply:
When you set on a RQ queue a timeout, this parameter gets injected into worker jobs and I would not like to handle this with kwargs since it's not self documenting. So I think this explicit approach is slightly better than kwargs.